src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -769,16 +770,16 @@ void EnSyatekiNiw_DrawEffects(EnSyatekiNiw* this, PlayState* play) {
     Gfx_SetupDL_25Xlu(play->state.gfxCtx);
 
     for (i = 0; i < EN_SYATEKI_NIW_EFFECT_COUNT; i++, effect++) {
-        if (effect->unk_00 == 1) {
+        if (effect->state == 1) {
             if (materialFlag == 0) {
                 gSPDisplayList(POLY_XLU_DISP++, gCuccoEffectFeatherMaterialDL);
                 materialFlag++;
             }
 
-            Matrix_Translate(effect->unk_04.x, effect->unk_04.y, effect->unk_04.z, MTXMODE_NEW);
+            Matrix_Translate(effect->pos.x, effect->pos.y, effect->pos.z, MTXMODE_NEW);
             Matrix_ReplaceRotation(&play->billboardMtxF);
-            Matrix_Scale(effect->unk_2C, effect->unk_2C, 1.0f, MTXMODE_APPLY);
-            Matrix_RotateZ(effect->unk_30, MTXMODE_APPLY);
+            Matrix_Scale(effect->scale, effect->scale, 1.0f, MTXMODE_APPLY);
+            Matrix_RotateZ(effect->rotZ, MTXMODE_APPLY);
```

Comment:
you can call this simply "rot"

With the billboarding (`Matrix_ReplaceRotation(&play->billboardMtxF);`),
the +z axis here corresponds to the axis orthogonal to and into the screen